assessment_software/src/app/table-component/table.component.css

```diff
@@ -0,0 +1,6 @@
+table{
+    @apply box-border border border-solid border-collapse;
+}
+td,th{
+    @apply box-border border border-solid text-left font-sans text-sm;
+}
```

Comment:
check how it behaves if you make a text very long. doesn't it overflow? wrap? how does it behave? 

![image](https://github.com/Aben-Bel/assessment_software/assets/44743368/672ef5a9-a8d8-465d-94ad-3d193963fd62)


is there a way to make the width fixed? rather than growing the table by size, what if it is scrollable?

Reply:
the text gets wrapped, it doesn't overflow. and it's 'scrollable' now
![image](https://github.com/Aben-Bel/assessment_software/assets/99379453/1f2b1e26-2098-45c1-a5ee-e4ba9fa7a73e)